### DISCUSSION TITLE
How to stop MPV from scanning the entire folder when opening files?

### DISCUSSION THREAD

**Discussion by KnIfER** [mpv-player/mpv#14584] [Q&A] [answered]
libmpv seems to have such an issue.  The entire folder was scanned for a full list of files in that directory. 

**If you switch to a different player kernel, you won't have this behavior.**

( I've used  logs of dokany/FUSE to inspect any file access )

---

I even downloaded potplayer to make sure that such a problem really existed.

In PotPlayer, even if "Add only selected files to playlist" is set, opening one single video file by pasting the full path of it will trigger a active scan inside of the folder.

---

Note: I'm not a potplayer user, I'm just using libmpv to implement a video player, and I found this problem：when the directory contains a large number of files, switching files will become slow due to extra scanning. 

`mpv_set_property_string(mpv, "sub-auto", "no");` is already set

What exactly caused my program to scan the entire folder (when the player component is set to libmpv) ?

**Comment by guidocella** [comment, marked as answer]
`--autoload-files=no`
